.gitmodules

```diff
@@ -1,11 +1,7 @@
-[submodule "evmjit"]
-	path = evmjit
-	url = https://github.com/ethereum/evmjit
-	branch = develop
 [submodule "cmake/Hunter/disabled-mode"]
 	path = cmake/Hunter/disabled-mode
-	url = https://github.com/hunter-packages/disabled-mode
+	url = https://github.com/hunter-packages/disabled-mode.git
```

Comment:
I heard GitHub is removing the `.git` alias.

Reply:
Weird, I have never heard of it. I can revert that part though. It's just that traditionally I always use `.git` myself. :)